cmake/detray-compiler-options-cpp.cmake

```diff
@@ -4,52 +4,57 @@
 #
 # Mozilla Public License Version 2.0
 
-# Include the helper function(s).
-include(detray-functions)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
Could you also update this in the README?

Reply:
Done, thanks!